org.hl7.fhir.utilities/src/main/resources/rendering-phrases_de.properties

```diff
@@ -4,3 +4,5 @@ BUNDLE_IF_NONE = If-None-Exist = {0}
 BUNDLE_LOCATION = Location = {0}
 BUNDLE_RESOURCE = Resource {0}:
 BUNDLE_SEARCH = Suche:
+CODESYSTEM_CONTENT_NOTPRESENT = Ich bin ein auslander

```

Comment:
I really think we better fix these before we accept the PR

Reply:
Good catch. Left over from testing. There are things here that should be fixed, but this at least addresses the originally reported issue.

I will leave test cases out of this PR for now, as the correct behaviour of rendering phrases, plural rules, and default locales are out of the scope of the original issue.